flink/src/test/java/org/apache/iceberg/flink/TestFlinkCatalogDatabase.java

```diff
@@ -59,6 +59,7 @@ public void testCreateNamespace() {
   @Test
   public void testDefaultDatabase() {
     sql("USE CATALOG %s", catalogName);
+    sql("show tables");
```

Comment:
Nit: other statements use all caps for SQL reserved words. Should this be `SHOW TABLES`?